backend/app/main.py

```
@@ -679,6 +679,17 @@ class VideoRequest(BaseModel):
     output_folder: str
     transcription_model: str = "faster-whisper"
     use_groq: Optional[bool] = None
+    target_language: Optional[str] = None
+    task: Literal["transcribe", "translate"] = "transcribe"
+
+    @field_validator("target_language")
+    @classmethod
+    def validate_target_language_bcp47(cls, v):
```

Ruff (0.16.0)

[warning] Missing return type annotation for classmethod `validate_target_language_bcp47`

(ANN206)